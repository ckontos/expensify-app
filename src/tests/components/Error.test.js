import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/Error';

test('should render the error page correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});
