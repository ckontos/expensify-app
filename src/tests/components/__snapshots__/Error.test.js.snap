// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the error page correctly 1`] = `
<div>
  404 error - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
